java/vector/src/main/java/org/apache/arrow/vector/complex/ListVector.java

```diff
@@ -710,14 +710,29 @@ public Object getObject(int index) {
   /**
    * Check if element at given index is null.
    *
-   * @param index  position of element
+   * @param index position of element
    * @return true if element at given index is null, false otherwise
    */
   @Override
   public boolean isNull(int index) {
     return (isSet(index) == 0);
   }
 
+  /**
+   * Check if element at given index is empty list.
+   * @param index position of element
+   * @return true if element at given index is empty list, false otherwise
+   */
+  public boolean isEmpty(int index) {
```

Comment:
@Override I'm surpised Error prone didn't can this.

Reply:
Thanks, fixed.